tests/test_preupg.py

```diff
@@ -273,95 +274,125 @@ def test_solution_www_bold_tag(self):
-            "profile": "xccdf_preupg_profile_default",
-            "result_dir": self.temp_dir,
-            "skip_common": True,
-            "temp_dir": self.temp_dir,
-            "id": None,
-            "debug": True,  # so root check won't fail
-        }
-
-        dc = DummyConf(**conf)
-        cli = CLI(["--contents", "tests/FOOBAR6_77/dummy_preupg/all-xccdf.xml"])
-        a = Application(Conf(dc, settings, cli))
-        # Prepare all variables for test
-        a.conf.source_dir = os.getcwd()
-        a.content = a.conf.contents
-        a.basename = os.path.basename(a.content)
-        self.assertEqual(a.get_scenario(), "FOOBAR6_77")
-
-    def test_migration_content_scenario(self):
-        """
-        Basic test for whole program
-        """
-        conf = {
-            "contents": "tests/FOOBAR6_CENTOS6/dummy_preupg/all-xccdf.xml",
-            "profile": "xccdf_preupg_profile_default",
-            "result_dir": self.temp_dir,
-            "skip_common": True,
-            "temp_dir": self.temp_dir,
-            "id": None,
-            "debug": True,  # so root check won't fail
-        }
-
-        dc = DummyConf(**conf)
-        cli = CLI(["--contents", "tests/FOOBAR6_CENTOS6/dummy_preupg/all-xccdf.xml"])
-        a = Application(Conf(dc, settings, cli))
-        # Prepare all variables for test
-        a.conf.source_dir = os.getcwd()
-        a.content = a.conf.contents
-        a.basename = os.path.basename(a.content)
-        self.assertEqual(a.get_scenario(), "FOOBAR6_CENTOS6")
-
-    def test_wrong_content_scenario(self):
-        """
-        Basic test for whole program
-        """
+        '''
+        Test to get right module directory with contents option,
+        directory with modules are the same where all-xccdf file is located
+        '''
         conf = {
-            "contents": "tests/FOOBAR6_7A/dummy_preupg/all-xccdf.xml",
-            "profile": "xccdf_preupg_profile_default",
-            "result_dir": self.temp_dir,
-            "skip_common": True,
-            "temp_dir": self.temp_dir,
-            "id": None,
-            "debug": True,  # so root check won't fail
+            "contents": "tests/Modules/all-xccdf.xml",
         }
-
         dc = DummyConf(**conf)
-        cli = CLI(["--contents", "tests/FOOBAR6_7A/dummy_preupg/all-xccdf.xml"])
+        cli = CLI(["--contents", "tests/Modules/all-xccdf.xml"])
         a = Application(Conf(dc, settings, cli))
         # Prepare all variables for test
         a.conf.source_dir = os.getcwd()
         a.content = a.conf.contents
         a.basename = os.path.basename(a.content)
-        self.assertEqual(a.get_scenario(), None)
-
-
-class TestPreupgradePrefix(base.TestCase):
-    def setUp(self):
-        settings.prefix = 'preupgrade'
-
-    def test_correct_prefix(self):
-        version = SystemIdentification.get_assessment_version('FOOBAR6_7')
+        self.assertEqual(a.get_scenario(), 'Modules')
+
+
+class TestValidScenario(base.TestCase):
```

Comment:
Fixed in https://github.com/upgrades-migrations/preupgrade-assistant/pull/233/commits/107a550a8ec5aeb8ddcbd5e59fef0d6a99e3e0a2